dashboard/lib/widgets/task_grid.dart

```diff
@@ -71,15 +71,15 @@ class TaskGridContainer extends StatelessWidget {
 /// Results are displayed in a matrix format. Rows are commits and columns
 /// are the results from tasks.
 class TaskGrid extends StatefulWidget {
-  const TaskGrid({
-    Key? key,
-    // TODO(ianh): We really shouldn't take both of these, since buildState exposes status as well;
-    // it's asking for trouble because the tests can (and do) describe a mutually inconsistent state.
-    required this.buildState,
-    required this.commitStatuses,
-    this.filter,
-    this.useAnimatedLoading = false,
-  }) : super(key: key);
+  const TaskGrid(
+      {Key? key,
+      // TODO(ianh): We really shouldn't take both of these, since buildState exposes status as well;
+      // it's asking for trouble because the tests can (and do) describe a mutually inconsistent state.
+      required this.buildState,
+      required this.commitStatuses,
+      this.filter,
+      this.useAnimatedLoading = false})
```

Comment:
nit: Prefer trailing commas

Reply:
Yeah, sorry that is a really hard habit for me to break.

Reply:
No worries, it would be nice if we added https://dart-lang.github.io/linter/lints/require_trailing_commas.html to the analysis options for the project

Reply:
Fixed the super.key and trailing commas. Let me look into adding that to analysis.

Reply:
Looks like the trailing comments was commented out in the flutter project due to an issue but is now closed. I can certainly enable it but I am not sure. I'll defer to you on this one.

Reply:
Let's add it. If it's too much of a diff for this PR, feel free to create a TODO to follow up with

Reply:
It's about 82 changes. I'll do it but won't include it with this change. I'll make a follow up one and just submit after this.